services/processor/native/get_code.go

```diff
@@ -52,21 +52,26 @@ func (s *service) retrieveContractInfo(ctx context.Context, executionContextId p
 func (s *service) retrieveDeployedContractInfoFromState(ctx context.Context, executionContextId primitives.ExecutionContextId, contractName string) (*sdkContext.ContractInfo, error) {
 	start := time.Now()
 
-	codeBytes, err := s.callGetCodeOfDeploymentSystemContract(ctx, executionContextId, contractName)
+	codeBytes, err := s.getFullCodeOfDeploymentSystemContract(ctx, executionContextId, contractName)
```

Comment:
i'd change the codeBytes var name to reflect its multi files

Reply:
Fixed